backend/libbackend/sql_compiler.ml

```diff
@@ -85,28 +85,30 @@ let binop_to_sql (op : string) : tipe_ * tipe_ * tipe_ * string =
       error2 "This function is not yet implemented" op
 
 
-let unary_op_to_sql op : tipe_ * tipe_ * string * string list =
+let unary_op_to_sql op : tipe_ * tipe_ * string * string list * bool =
```

Comment:
We try to avoid `bool`s as arguments, and use variants instead. So here you could have a type

```
type position = Front | Back
```

And then return `(TBool , TBool, "bot", [], Back)`

Reply:
Sounds good! Any advice on an appropriate place to define the type? Not sure I understand the right context to put it in.